FirebaseRemoteConfig/Sources/RCNConfigSettings.m

```diff
@@ -32,6 +32,9 @@
 static NSString *const kRCNUserDefaultsKeyNameLastSuccessfulFetchTime = @"lastSuccessfulFetchTime";
 static const int kRCNExponentialBackoffMinimumInterval = 60 * 2;       // 2 mins.
 static const int kRCNExponentialBackoffMaximumInterval = 60 * 60 * 4;  // 4 hours.
+static NSString *const RCNExternalUserDefaultsKeyLastSuccessfulFetchTime =
```

Comment:
Nit: should still have a `k` prefix to match existing constants.

Reply:
had got fb in a previous CL to not add k along with the prefix (style guide)